### PR TITLE
Serialize SCON:   added implementation of `serialize_SCON`

### PR DESCRIPTION
**Pull request by Oddan**
NB: The value 'Kh' (effective for connection) is not directly available from the Connection interface.  For the time being, a dummy value is used.

Computing the _correct_ value for 'Kh' involves using the connection transmissibility factor (avaliable), the well diameter (available), as well as information about grid geometry (not immediately available), well angle through connection cell (not immediately available, as far as I can tell), and possibly a conversion factor.

### CODE REVIEW

File: src/opm/output/eclipse/WellDataSerializers.cpp [hunk 1]
```diff
@@ -22,10 +22,40 @@
...
+    size_t well_offset = 0;
+    for (const Opm::Well* well : sched_wells) {
+        const auto& completions = well->getCompletions( lookup_step );
+        size_t completion_offset = 0;
+        for (const auto& completion : completions) {
+
+            const size_t offset = well_offset + completion_offset;
+
+            data[ offset + SCON_CF_INDEX ] =
+                completion.getConnectionTransmissibilityFactor();
```
Comment (joakim-hove) [inline src/opm/output/eclipse/WellDataSerializers.cpp]: Somehow we need to get the value *actually used by the simulator* passed in here; we should **not** even think about calculating this with cell geometries and what not in the output layer. 

There is already functionality in the output for taking various quantities calculated by the simulator as "extra" (see also: https://github.com/OPM/opm-common/pull/391). I guess getting prope `CF` values should be solved something like that, or alternatively we can update the connection factors in the `Schedule` object - but that is ugly.

Comment (Oddan) [reply src/opm/output/eclipse/WellDataSerializers.cpp]: To my understanding, the value at SCON_CF_INDEX should be the transmissibility factor, so this value should be OK here.  The problem is the value at SCON_KH_INDEX, which I for the time being have flagged with UNIMPLEMENTED_VALUE.

Comment (joakim-hove) [reply src/opm/output/eclipse/WellDataSerializers.cpp]: Ok - I don't have the details in my head, but main point still stands, no trivial values should come from the simulator.

**Comment by atgeirr** [comment]
Superseded by #400, closing.

**Pull request closed without merging**